src/cljs/pyregence/pages/dashboard.cljs

```diff
@@ -141,9 +141,9 @@
      [:td ; "Delete"
       [:div {:style {:display "flex" :justify-content "center"}}
        [icon-button :trash
-                    #(handle-delete-match-drop match-job-id display-name)
-                    nil
-                    :btn-size :circle]]]]))
+        #(handle-delete-match-drop match-job-id display-name)
+        nil
+        :btn-size :circle]]]]))
```

Comment:
Let's move `:trash` down to its own line. This matches what we have in various other places of the codebase. For example, in `near_term_forecast.cljs`:
```clojure
              [information-tool
               get-point-info!
               @my-box
               select-layer-by-hour!
               (get-current-layer-key :units)
               (get-current-layer-key :convert)
               (or (get-current-layer-key :no-convert) #{})
               (get-current-layer-hour)
               #(set-show-info! false)])
```